### DISCUSSION TITLE
posts/friends

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#21] [General]
# posts/friends

Friends of whexy.

http://localhost:3000/friends

**Comment by whexy** [comment]
### Data Source
- Names and avatars are retrieved from GitHub, revalidated every 600 seconds.
- Recent updates are fetched from RSS source, revalidated every 600 seconds.

### Story
- A ring lights up when a friend publishes at least one post within 30 days.
- Scroll the name bar horizontally to view more.

This page is (obviously) following the design of Instagram, which is a famous social media App from Meta.

**Comment by Joycoooi** [comment]
这真是大佬，从这个网站学习到了很多知识，谢谢您的无私分享！
